### DISCUSSION TITLE
"Incoming" listed twice for resources in inventory

### DISCUSSION THREAD

**Discussion by cbortz** [pwillworth/galaxyharvester#23] [Q&A] [answered]
When I add a resource to my inventory, it gives me two options, but they are both "Incoming". Is this intended? How should I be interpreting this?

![image](https://user-images.githubusercontent.com/184307/114470334-b3f0ed00-9bbc-11eb-99bf-0a37c36cc230.png)

**Comment by pwillworth** [comment, marked as answer]
"Incoming" is always present at the top of the list to be a default incase the person does not have any groups and to avoid defaulting to the alphabetical first one they have if they do.  In your case, you have previously moved a resource to the Incoming group, but never moved it to or created any different groups.  So, you just see the default that is always there, plus your newly created group named "Incoming".

**Reply by cbortz** [reply]
Thanks for the explanation!
